javascript/ql/src/experimental/Security/CWE-340/TokenBuiltFromUUID.ql

```diff
@@ -0,0 +1,51 @@
+/**
+ * @name Predictable token
+ * @description Tokens used for sensitive tasks, such as, password recovery,
+ *  and email confirmation, should not use predictable values.
+ * @kind path-problem
+ * @precision medium
+ * @problem.severity error
+ * @security-severity 5
+ * @id py/predictable-token
+ * @tags security
+ *       external/cwe/cwe-340
+ */
+
+import javascript
+import DataFlow
+import DataFlow::PathGraph
+
+class PredictableResultSource extends DataFlow::Node {
+  PredictableResultSource() {
+    exists(API::Node uuidCallRet |
+      uuidCallRet =
+        API::moduleImport("uuid").getMember(["v1", "v2", "v3", "v5"]).getACall().getReturn()
```

Comment:
```suggestion
        API::moduleImport("uuid").getMember(["v1", "v3", "v5"]).getACall().getReturn()
```
Same fix as the Python query.  

Otherwise this looks fine for an experimental query.

Reply:
This change caused the autoformatter to become unhappy (probably because the line is now shorter, and so it can format things in a different way). 

@bananabr Could you reformat this file and push the formatted version? 🙂 
(I think that's all that's needed in order for us to merge it.)

Reply:
Done. I also took the chance to remove the unnecessary getACAll call from the JS query.